taipy/templates/default/hooks/post_gen_project.py

```diff
@@ -77,33 +77,29 @@ def handle_single_page_app():
         main_file.write("\n")
         main_file.write("    gui = Gui(page=page)\n")
 
+    with open(os.path.join(os.getcwd(), "sections", "import.txt"), "a") as import_file:
+        import_file.write("import taipy.gui.builder as tgb\n")
+
     handle_run_service()
 
     with open(os.path.join(os.getcwd(), "sections", "page_content.txt"), "a") as page_content_file:
         page_content_file.write(
-            '''
-page = """
-<center>
-<|navbar|lov={[("home", "Homepage")]}|>
-</center>
+            """
+with tgb.Page() as page:
```

Comment:
Normally we use:
```python
with tgb.part(class_name="text-center"):
        tgb.button("Home")
```
to center elements in tgb. But this does not work with navbar for some reason.

Reply:
Since navbar is a div I center it by doing:
```python3
with tgb.part():
    tgb.navbar()
```

and then applying the following css to the taipy-navbar class:
```css
.taipy-navbar {
    max-width: fit-content;
    margin-left: auto;
    margin-right: auto;
}
```

Reply:
Looks good. Thx.